packages/rds-signer/package.json

```diff
@@ -60,9 +60,7 @@
   "browser": {
     "./dist-es/runtimeConfig": "./dist-es/runtimeConfig.browser"
   },
-  "react-native": {
-    "./dist-es/runtimeConfig": "./dist-es/runtimeConfig.native"
```

Comment:
Is this removal on purpose? Does React Native has any other bundlers which might require this configuration?

If not, can we remove the `react-native` key itself?

Reply:
this removal is on purpose because you asked that we do not explicitly add the replacements that would be automatic. 

react-native is left as an object because of the consistency enforcement that `browser` and `react-native` are of the same type: object or string or undefined.

this prevents errors like forgetting to add the react-native directive when creating a browser replacement